modules/mod_ginger_rdf/mod_ginger_rdf.erl

```diff
@@ -34,18 +34,8 @@
 -record(state, {context}).
 
 manage_schema(_, Context) ->
-    Datamodel = #datamodel{
-        categories=[
-            {rdf, meta, [{title, <<"RDF resource">>}]}
-        ],
-        resources=[
-            {rdf_content_group, content_group, [
-                {title, <<"RDF resources">>}
-            ]}
-        ]
-    },
-    z_datamodel:manage(?MODULE, Datamodel, Context),
-
+    %% TODO: Legacy code ahead! This function must not have side effects;
+    %% it should only return the data model.
```

Comment:
We should look into this case to see if it is still necessary to keep this here, or if it could be in a better place.

Reply:
Use `manage_data/2` to update/install data after the schema has been installed.